app/src/main/java/com/mathewsachin/fategrandautomata/accessibility/ScriptRunnerUserInterface.kt

```diff
@@ -112,30 +124,26 @@ class ScriptRunnerUserInterface(val Service: ScriptRunnerService) {
     }
 
     private fun getMaxBtnCoordinates(): Location {
-        val rotation = windowManager.defaultDisplay.rotation
-        val rotate = rotation == Surface.ROTATION_0 || rotation == Surface.ROTATION_180
+        val m = metrics
 
-        val x = (if (rotate) {
-            metrics.heightPixels
-        } else metrics.widthPixels) - layout.measuredWidth
-        val y = (if (rotate) {
-            metrics.widthPixels
-        } else metrics.heightPixels) - layout.measuredHeight
+        val x = m.widthPixels - layout.measuredWidth
```

Comment:
Can you explain this logic? Is `layout` only the button? So if the screen is 1280 pixels wide and the button is 100 pixels wide, x == 1180?

Please add KDoc to this method.

Reply:
Yup, just as you thought. I don't think it was even complicated. The code before this PR was of course. I added KDoc, and also changed the name of `layout` to `scriptCtrlBtnLayout`.